### ISSUE TITLE
Error in sfepy/script/convert_mesh.py while following example from the documentation

### ISSUE THREAD

**Issue by jan-heck** [sfepy/sfepy#607]
I recently found sfepy and am impressed by the great documentation and the project in general! :)

I was able to successfully set everything up and go through http://sfepy.org/doc-devel/examples/diffusion/cube.html#diffusion-cube without problems.

Now I wanted to work on my own geometries and was following the example given in http://sfepy.org/doc-devel/preprocessing.html . I am following the openscad workflow. Both with my own geometry, and simply following the example given, I run into the same error `KeyError: ('vertex', 3)` when converting the `.msh` into `.vtk` using a script by sfepy:

```
$ ~/git/sfepy/script/convert_mesh.py screwdriver_handle.msh screwdriver_handle.vtk

sfepy: reading mesh (screwdriver_handle.msh)...
Traceback (most recent call last):
  File "/home/arch/git/sfepy/script/convert_mesh.py", line 237, in <module>
    main()
  File "/home/arch/git/sfepy/script/convert_mesh.py", line 162, in main
    mesh = Mesh.from_file(filename_in)
  File "/home/arch/git/sfepy/sfepy/discrete/fem/mesh.py", line 220, in from_file
    mesh = io.read(mesh, omit_facets=omit_facets)
  File "/home/arch/git/sfepy/sfepy/discrete/fem/meshio.py", line 376, in read
    cell_types.append(self.cell_types[(c.type, dim)])
KeyError: ('vertex', 3)
```

I am happy for any hints as to what could be going wrong. Thanks! :)

**Comment by rc** [comment]
Thanks for reporting the issue! We have recently started to use the meshio package for reading/writing meshes, so this problem is probably related. @vlukes, would you mind looking at this and eventually update the preprocessing tutorial?

**Comment by jan-heck** [comment]
Hi everyone, I am keen to get started with using sfepy for my research. Thank you for looking into the issue. Meanwhile, I am wondering if there is some workaround for the issue; or how you are generating (meshing) your geometries at the moment?

**Comment by rc** [comment]
Hi @jan-heck I have just merged #614 (thanks to @vlukes) , which should fix the problem. Get the latest git version, and run:
```
~/git/sfepy/script/convert_mesh.py -d 3 screwdriver_handle.msh screwdriver_handle.vtk
```
Let us know if that helps, please - if not, reopen this issue.
